test_ex_17_v2.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+import unittest
+from ex_17_v2 import calculate_male_bac, calculate_female_bac, determine_person_to_drive_legally
+
+class MaleBacTest(unittest.TestCase):
+    """Test for calculating male BAC"""
+
+    def test_male_bac(self):
+        """Calculate male bac"""
+
+        alcohol_amount = 10
+        weight = 130
+        hours_since_last_drink = 1
+
+        male_bac_calculation = calculate_male_bac(alcohol_amount, weight, hours_since_last_drink)
+        self.assertEqual(male_bac_calculation, '0.29')
+
+unittest.main()
+
+class FemaleBacTest(unittest.TestCase):
+    """Test for female bac"""
+
+    def test_female_bac(self):
+        """Calculate female bac"""
+
+        alcohol_amount = 10
+        weight = 130
+        hours_since_last_drink = 1
+
+        female_bac = calculate_female_bac(alcohol_amount, weight, hours_since_last_drink)
+        self.assertEqual(calculate_female_bac, 0.26)
+
+unittest.main()
+
+
+class DetermineIfLegallyDrive(unittest.TestCase):
```

Comment:
@paulghaddad Do I have this test written correctly? It was tricky b/c the function it's testing has an if-else.